frontend/src/views/dev/Performance.vue

```diff
@@ -69,15 +78,10 @@ export default {
 </script>
 
 <style scoped lang="scss">
-.result-table {
-  border: 1px black solid;
-  border-collapse: collapse;
-  tr {
-    td,
-    th {
-      border: 1px black solid;
-      padding: 5px;
-    }
+::v-deep() th[role='columnheader'].text-right {
+  direction: rtl;
+  span {
+    direction: ltr;
```

Comment:
this looks strange, why do we need rtl, and then switch back to ltr in the inner span?
isn't this just right text alignment?

Reply:
This ensures that the column heading text is aligned to the right (just like the content), and the up/down arrow is on the left side.